integration_tests/build_test_containers.sh

```diff
@@ -1,11 +1,22 @@
 #!/usr/bin/env bash
 
-if [[ "$(docker images -q d3a-tests 2> /dev/null)" == "" ]]; then
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
+    docker_command="sudo docker"
+else
+    docker_command="docker"
+fi;
+
+D3A_IMAGE_TAG="d3a-tests"
+
+if [[ "$(${docker_command} images -q ${D3A_IMAGE_TAG} 2> /dev/null)" == "" ]]; then
+  echo "Building d3a image ..." && \
+  rm -rf tests/d3a && \
```

Comment:
Why do you try to delete the cloned repo again ? Does not hurt of course, since rm -rf does not raise an error if the file/directory is not there, but still it could be avoided IMO

Reply:
hach, yes, you are right. Will revert it then. Is a remnant of me trying to make it run on my linux machine.

Reply:
If you are testing multiple times, the an Error (or rather a warning) is shown: "d3a folder already exists". Now that you are point it out, i dont know what git clone does with existing folders. So, i still vote for keeping it.